doc/source/whatsnew/v0.20.0.txt

```diff
@@ -1032,8 +1032,7 @@ Reshaping
 - Bug in ``pd.concat()`` in which concatting with an empty dataframe with ``join='inner'`` was being improperly handled (:issue:`15328`)
 - Bug with ``sort=True`` in ``DataFrame.join`` and ``pd.merge`` when joining on indexes (:issue:`15582`)
 
-Numeric
-^^^^^^^
```

Comment:
Agreed.  None of the changes should be in your PR given what you are focusing on.  Don't forget to add ones for your actual patches though!